### PR TITLE
Leicester Repeater Rebalance + New Sprite

### PR DESCRIPTION
**Pull request by KrabSpider**
## About The Pull Request
.45-70 Government Rifle Bullets. Exclusive to the ever degrading Leicester Repeater. Deals 70 damage with 20 AP and 1.25 sunder. 2 slowdown, 1 stagger within a 4 tile range. Ammo packets are available in venders. Also the repeater gets a new sprite

**PS: Fuck Ocelot, the generous creator of the sprite below.**
image

## Why It's Good For The Game
Something something gun bad
Something repeater lacking. 
Something easier to balance
Something something self defense

The repeater deserves better. Thats all. It is effectively a TX-15 with aimode that fires even worse TX15 slugs. It has no actual niche to fill beyond high damage sniping that a DMR and sniper can fill. Its DPS is practically outshined by **even the TX-11**. This gun has been done dirty. I *like* its existence and want to give it a chance before it gets shelved to the seasonal dimension. 

## Changelog
:cl:
balance: .45-70 Goverment Rifle Cartridges for the  leicester repeater. 70 damage/20 AP/1.25 sunder from 60 damage/ 20 AP/1.5 sunder. Bullets stagger and slow within a 4 tile range (from a weak slow at all ranges). 
imageadd: .45-70 Government Packets
imageadd: New Leicester Repeater sprite. Much thanks to SilentMP5A5, otherwise known by many as Ocelot.
/:cl:

### CODE REVIEW

**Comment by AndroidMK2** [comment]
Sprite looking very nice, would really like to see these changes implemented

**Comment by KrabSpider** [comment]
MJP I hate you so much. Anyway thank you

File: code/modules/projectiles/magazines/misc.dm [hunk 1]
```diff
@@ -115,3 +115,12 @@
 	current_rounds = 150
 	max_rounds = 150
 
+/obj/item/ammo_magazine/packet/p4570
+	name = "packet of .45-70"
+	desc = "A packet containing 50 rounds of .45-70 Government.."
```
Comment (XSlayer300) [inline code/modules/projectiles/magazines/misc.dm:120]:
```suggestion
	desc = "A packet containing 50 rounds of .45-70 Government."
```

**Review by BraveMole — CHANGES_REQUESTED** [review]
On the cl:
image
First should be balance, the two last are unneeded.

State was were the previous dp, sunder, etc of the gun

**Review by Kuro020 — APPROVED** [review]

**Review by BraveMole — APPROVED** [review]

**Pull request merged**